mmedit/datasets/sr_vimeo90k_multiple_gt_dataset.py

```diff
@@ -57,10 +56,14 @@ def load_annotations(self):
 
         data_infos = []
         for key in keys:
-            lq_paths = sorted(
-                glob.glob(osp.join(self.lq_folder, key, '*.png')))
-            gt_paths = sorted(
-                glob.glob(osp.join(self.gt_folder, key, '*.png')))
+            lq_paths = [
+                osp.join(self.lq_folder, key, f'im{i}.png')
+                for i in range(1, 8)
+            ]
+            gt_paths = [
+                osp.join(self.gt_folder, key, f'im{i}.png')
+                for i in range(1, 8)
```

Comment:
8 is a magic number. Can it be made into a variable at least?

Reply:
Yes it can be done. I will update this in the afternoon.